docs/GETTING_STARTED.md

```diff
@@ -1,288 +1,225 @@
-version` and `docker images` - both should succeed.
 
-**containerd** - to confirm that you have containerd working please check with
-`ctr version` and `ctr i ls`.  The version needs to be 1.4 or greater
+- **containerd >= 1.4** - to confirm that you have containerd working please check with
+`ctr version` and `ctr i ls`.
+- **ctr/nerdctl** - you need one of the containerd clients to interact with containerd/registry.
+- **fuse** - used for mounting without root access (`sudo yum install fuse`).
+- **zlib** - used for decompression and ztoc creation (`sudo yum install zlib-devel`).
+- **go** - the minimum supported go version is 1.18.
 
-**ctr** - to confirm that you have ctr, the containerd CLI installed, check with
-`ctr version` and `ctr i ls`
+## Install soci-snapshotter
 
-**soci-snapshotter** - this you will need to make sure you can build from
-source. You should be able to run `make` in the soci-snapshotter dir.
-Some common issues:
-- getting the github repository `git clone
-https://github.com/awslabs/soci-snapshotter.git`
-- having the zlib developer tools (zlib is used for decompression and ztoc creation).
-Often fixed with `sudo yum install zlib-devel`
-- having the proper go version installed, as of this document please use 1.18+
+The soci-snapshotter project consists of 2 main components:
 
-**fuse** - confirm FUSE is installed for mounting userspace filesystems with `fusermount -V`
-(optional, enables mounting without root access)
+- `soci`: the CLI tool used to build/manage SOCI indices.
+- `soci-snapshotter-grpc`: the daemon (a containerd snapshotter plugin) used for lazy loading.
 
-## Start a local ORAS registry
-Most OCI registries do not currently support the "referrers" feature.
-SOCI relies on this feature to link its indices and manifests.  The [ORAS
-project](https://oras.land/) tries to fill this gap; in their words:
+Many `soci` CLI commands need to be run as `sudo`, because the metadata is saved
+in directories that a non-root user often does not have access to.
 
-> Registries are evolving as generic artifact stores. To enable this goal, the ORAS project provides a way to push and pull OCI Artifacts to and from OCI Registries.
+Currently to get the binaries, we need to build the project from source after cloing the repo:
 
-[The longer term solution for this issue is using the OCI Reference
-Types](https://github.com/awslabs/soci-snapshotter/issues/42).  For now we are
-relying on the ORAS project for the referrers feature.
+```shell
+$ git clone https://github.com/awslabs/soci-snapshotter.git
+$ cd soci-snapshotter
+$ make
+```
 
-Given this we want to start our registry using an OCI image from the ORAS 
-project.  We will run it on port 5000 as an HTTP (**NOT** HTTPS) server without
-certificates.  To do this we will use Docker:
+This builds the project binaries into the `./out` directory. You can install them
+to a `PATH` directory (`/usr/local/bin`) with:
```

Comment:
Later examples require this, so I think this should be more insistent than "can".

Reply:
added a note (either `make install` or use full path `./out/soci`, the doc assumes they're installed).